src/DIRAC/Resources/Computing/BatchSystems/Host.py

```diff
@@ -104,7 +104,8 @@ def submitJob(self, **kwargs):
             stdout=subprocess.PIPE,
             shell=True,
             env=envDict,
-            universal_newlines=True)
+            universal_newlines=True,
```

Comment:
```suggestion
            universal_newlines=True
```

Reply:
Why are these commas added at the end of the Popen argument list ?

Reply:
To minimize the diff when the next person adds another argument

Reply:
Is it a new style recommendation ?

Reply:
It would be part of black, but we could also encourage it with flake8 https://github.com/PyCQA/flake8-commas